src/components/Molecules/Header/Nav/Nav.js

```diff
@@ -21,16 +21,20 @@ const MainNav = ({ navItems }) => {
   const [isExpandable, setIsExpandable] = useState(false);
   const [isSubMenuOpen, setIsSubMenuOpen] = useState({});
   const [isKeyPressed, setIsKeyPressed] = useState({});
+  let [width] = useState(null);
+  let [mobile] = useState(null);
+
+  useEffect(() => {
+    // Detect window screen size
+    width = window.innerWidth;
+    mobile = width < sizes.medium;
+  }, []);
 
   const toggleBurgerMenu = event => {
     event.preventDefault();
     setIsExpandable(!isExpandable);
   };
 
-  // Detect windown screen size
-  const width = window.innerWidth;
-  const mobile = width < sizes.medium;
-
   const toggleSubMenu = item => event => {
     if (mobile) {
```

Comment:
Nice job @s-barrah, currently toggleBurgerMenu is not working because mobile is set to null